dotBackendOnboarding.md

```diff
@@ -0,0 +1,227 @@
+	| test-postman | `just test-postman <collections>`| `./mvnw -pl :dotcms-postman verify -Dpostman.test.skip=false -Pdebug -Dpostman.collections={{ collections }}`  |
+
+	> ⚠️ Note that the command `just build-test-full` can consume a lot of machine resources and may take a long time to be completed. Its use is more recommended in the _**cloud environment**_.
+
+	> 💡**Tip**:  If you don't provide any collections the command `just test-postman` will run **page** collection tests by default.
+
+## Debugging
+
+Debugging is essential for diagnosing and resolving issues that arise during development. This section will guide you through the debugging process using IntelliJ IDEA. 
+
+With the latest just file changes introduced, there is an easier way to debug **dotCMS**, and also you can run IT using the IDE. Please be aware that since everything is running in Docker, you might need to keep an eye on the containers, volumes and images in Docker Hub.
+
+### Debug mode
+
+It will start dotCMS debug mode without any other configuration required, and the port debug will be **_5005_**. All will be running in **Docker** (**Elasticsearch**, **PostgresSQL** and **dotCMS**).
+
+1. **First, we need to build the project**:
+	| Action | Just | Maven |
+	|--|--|--|
+	| build | `just build` | `./mvnw -DskipTests clean install`
+
+2. **And then run this one**:
+	| Action | Just | Maven |
+	|--|--|--|
+	| Startup | `just dev-run-debug-suspend` | `./mvnw  -pl  :dotcms-core  -Pdocker-start,debug-suspend`
+
+### Integration tests using IDE
+
+1. **First, we need to build the project**:
+	| Action | Just | Maven |
+	|--|--|--|
+	| Build | `just build` | `./mvnw -DskipTests clean install`
+
+2. **Then, we need to start up the following service**:
+	| Action | Just | Maven |
+	|--|--|--|
+	| Start | `just test-integration-ide` | `./mvnw -pl :dotcms-integration pre-integration-test -Dcoreit.test.skip=false`
+
+	> ℹ️ **Note:** This command prepares  the  environment  for  running  integration  tests  in  an  IDE.
+
+3. After that, you can click **Debug / Run** directly on the IDE for the test class you want to.
+4. **Once testing is finished**, you can stop the test integration services through the following command:
+	| Action | Just | Maven |
+	|--|--|--|
+	| Stop | `just test-integration-stop` | `./mvnw -pl :dotcms-integration -Pdocker-stop -Dcoreit.test.skip=false`
+
+## Advanced Configurations
+
+Advanced configurations allow you to customize and optimize your development environment and application settings. This section covers configuring external resources like **PostgreSQL** and **Elasticsearch**, customizing Maven build options, and adjusting **Docker** settings. By tailoring these configurations to your specific needs, you can enhance performance, improve security, and streamline your development workflow.`
+
+### dotCMS application settings
+
+**Configure initial admin password**:
+
+* Set up the following env variables in the `~/.zshrc` file.
```

Comment:
This should not be needed any more unless using some other runtime configuration.   The maven build automatically sets the initial password when needed.   This would only be used when running directly on an external tomcat.  The docker-compose also set the prameter.